### PR TITLE
release 1.1.1

### PR DESCRIPTION
**Pull request by Profpatsch**
I’d like to push the manpage change to nixpkgs.

The MAINTAINERS says we don’t use the patch version field, but I don’t think it’s wise to bump the minor version for an update in documentation, so I’d use the patch field in this case.

### CODE REVIEW

**Comment by nyarly** [comment]
This LGTM. A man page would be very nice to have and have released.

**Comment by curiousleo** [comment]
> The MAINTAINERS says we don’t use the patch version field, but I don’t think it’s wise to bump the minor version for an update in documentation, so I’d use the patch field in this case.

I see no harm in increasing the minor version. But if you want to do it like you said, go ahead. Before you do, please change MAINTAINERS accordingly to describe under which circumstances a change warrants a patch version increase. MAINTAINERS is useless if it doesn't reflect actual practice, and releasing a patch version currently contradicts what MAINTAINERS says.

**Comment by Profpatsch** [comment]
I’m gonna ignore the clippy failure, it’s unrelated and will pop up in the next PR.

File: Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name = "lorri"
 # lorri uses a MAJOR.MINOR versioning scheme, see MAINTAINERS.md for details.
 # Cargo requires MAJOR.MINOR.PATCH. So we translate lorri's versioning scheme
 # to MAJOR.MINOR.0, i.e. the PATCH part is always zero.
```
Comment (curiousleo) [inline Cargo.toml:5]: Please update or remove this comment, it is now out of date.

**Pull request merged**